tests/clv/models/test_beta_geo.py

```diff
@@ -160,63 +118,133 @@ def test_expected_probability_alive(self):
         The "true" prefix refers to the value obtained using 1) the closed form
         solution and 2) the data-generating parameter values.
         """
-        true_prob_alive = self.alive.mean()  # scalar
-        est_prob_alive = self.fixed_model.expected_probability_alive(
-            self.customer_id,
-            self.frequency,
-            self.recency,
-            self.T,
+        rng = np.random.default_rng(152)
+
+        N = 100
+        # Almost deterministic p = .02
```

Comment:
What does an almost deterministic dropout probability here check?

Reply:
That the probability of dropout is stable at a value that should induce a nice `p_alive` ~ 0.5, nothing too low nor too high

Reply:
Added a comment